internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -34,15 +33,14 @@ import (
 	"github.com/decred/dcrd/dcrutil/v3"
 	"github.com/decred/dcrd/gcs/v2"
 	"github.com/decred/dcrd/internal/mempool"
-	"github.com/decred/dcrd/internal/rpcserver"
 	"github.com/decred/dcrd/internal/version"
 	"github.com/decred/dcrd/peer/v2"
 	"github.com/decred/dcrd/rpc/jsonrpc/types/v2"
 	"github.com/decred/dcrd/wire"
 )
 
 // testDataPath is the path where all rpcserver test fixtures reside.
-var testDataPath = filepath.Join("internal", "rpcserver", "testdata")
+var testDataPath = filepath.Join("testdata")
```

Comment:
This can just be `var testDataPath = "testdata"` now since there is no joining going on.